### PR TITLE
Updates to configuration and relevant code

### PR DESCRIPTION
**Pull request by dev-kiranbanda**
- Fixed the FulfillmentApiBaseURL in appSettings.json
- Updated the URL helper to prepend "/" while building the URLs for Fulfillment and metering apis

### CODE REVIEW

File: src/Microsoft.Marketplace.SaaS.SDK.Client/Helpers/UrlHelper.cs [hunk 1]
```diff
@@ -20,7 +20,7 @@ public static string GetSaaSApiUrl(SaaSApiClientConfiguration clientConfiguratio
         {
             var resourceId = Convert.ToString(resourceGuid);
             string operationId = string.Empty;
-            string subscriptionBaseURL = "saas/subscriptions/";
+            string subscriptionBaseURL = "/saas/subscriptions/";
```
Comment (code4clouds) [inline src/Microsoft.Marketplace.SaaS.SDK.Client/Helpers/UrlHelper.cs]: This is also ending with a bar... the bar needs to be carry over to the code that implements the subscriptionBaseURL.

Comment (dev-kiranbanda) [reply src/Microsoft.Marketplace.SaaS.SDK.Client/Helpers/UrlHelper.cs]: Sure, however this is internal implementation so should not affect user / configuration

Comment (code4clouds) [reply src/Microsoft.Marketplace.SaaS.SDK.Client/Helpers/UrlHelper.cs]: Agree... but its best practice for REST APIs.  Also the variable should be subscriptionBasePath as is not longer a URL.  Not picking the code... just keep them in mind in the future.

Comment (dev-kiranbanda) [reply src/Microsoft.Marketplace.SaaS.SDK.Client/Helpers/UrlHelper.cs]: Sure, thank you for the inputs, taken a note of it.

**Pull request merged**